kolibri/plugins/coach/assets/src/views/common/QuizStatus.vue

```diff
@@ -0,0 +1,248 @@
+          return new Date(this.exam.date_activated);
+        } else {
+          return null;
+        }
+      },
+    },
+    methods: {
+      handleOpenQuiz() {
+        let promise = ExamResource.saveModel({
+          id: this.$route.params.quizId,
+          data: {
+            active: true,
+            date_activated: new Date(),
+          },
+          exists: true,
+        });
+
+        return promise
+          .then(() => {
+            this.$store.dispatch('classSummary/refreshClassSummary');
+            this.showConfirmationModal = false;
+            this.$store.dispatch('createSnackbar', this.coachString('quizOpenedMessage'));
+          })
+          .catch(() => {
+            this.$store.dispatch('createSnackbar', this.coachString('quizFailedToOpenMessage'));
+          });
+      },
+      handleCloseQuiz() {
+        let promise = ExamResource.saveModel({
+          id: this.$route.params.quizId,
+          data: {
+            archive: true,
+            date_archived: new Date(),
+          },
+          exists: true,
+        });
+
+        return promise
+          .then(() => {
+            this.$store.dispatch('classSummary/refreshClassSummary');
+            this.showCancellationModal = false;
+            this.$store.dispatch('createSnackbar', this.coachString('quizClosedMessage'));
+          })
+          .catch(() => {
+            this.$store.dispatch('createSnackbar', this.coachString('quizFailedToCloseMessage'));
+          });
+      },
+      handleToggleVisibility() {
+        const newActiveState = !this.exam.active;
+        const snackbarMessage = newActiveState
+          ? this.coachString('quizVisibleToLearners')
+          : this.coachString('quizNotVisibleToLearners');
+
+        let promise = ExamResource.saveModel({
+          id: this.$route.params.quizId,
+          data: {
+            active: newActiveState,
+          },
+          exists: true,
+        });
+
+        return promise.then(() => {
+          this.$store.dispatch('classSummary/refreshClassSummary');
+          this.showConfirmationModal = false;
+          this.$store.dispatch('createSnackbar', snackbarMessage);
+        });
+      },
+    },
+    $trs: {
+      quizClosedLabel: {
+        message: 'Quiz closed',
+        context:
+          'A label indicating that the currently viewed quiz is closed - meaning that learners may no longer give answers to the quiz.',
+      },
+      reportVisibleToLearnersLabel: {
+        message: 'Report visible to learners',
+        context:
+          'The label for a switch that will toggle whether or not learners can view their quiz report.',
+      },
+      questionOrderLabel: {
+        message: 'Question order',
+        context: 'A label for the place where the question order is shown.',
+      },
+      yesLabel: {
+        message: 'Yes',
+        context: "Used to indicate to the user when a switch element has a true or 'on' value.",
+      },
```

Comment:
Indicates the value of the switch element turned 'on', and the quiz report is visible for the learners.

Reply:
Shouldn't there be a `No` value here, instead of `Yes`? Toggle is turned off... :confused: 

![LEDev1911n  Running  - Oracle VM VirtualBox_023](https://user-images.githubusercontent.com/1457929/67449510-b2271880-f61a-11e9-81fb-f0c8ba3b098a.png)

`<rant-on>`
_I really, **really**, dislike toggles... Nothing but confusion_ :unamused: 
`<rant-off>`